bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/Strategy.scala

```diff
@@ -84,9 +86,9 @@ object Strategy {
       val ao = ctx.getAOComponent.asScalaOpt
       if (ao.exists(_.isInstanceOf[InstAltair])) {
         ao.get.asInstanceOf[InstAltair].getMode match {
-          case AltairParams.Mode.LGS_P1 => List(Pwfs1North)
+          case AltairParams.Mode.LGS_P1 => List(Pwfs1North, GmosNorthOiwfsPwfs1)
```

Comment:
It *is* possible to use LGS+P1.  The LGS + PWFS1 options are:
GMOS + LGS using PWFS1 for TTF without flexure compensation (AGS selects PWFS1 only)
GMOS + LGS using PWFS1 for TTF and OIWFS for flexure compensation (AGS selects PWFS1 + OIWFS)